packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.tsx

```diff
@@ -171,6 +186,26 @@ export class GeovEntityPropsByPredicate {
     }
   }
 
+  componentDidLoad() {
+    // Display Expand button if need. The page must be loaded in order to have the measurements
+    const shadowRoot = this.hostElement.shadowRoot;
+    if (shadowRoot) {
+      const itemLitteral = shadowRoot.querySelector('.litteral-container .litteral-collapsed');
+      const itemButton = shadowRoot.querySelector('.litteral-container .item-button') as HTMLElement;
+
+      if (itemLitteral && itemButton) {
+        itemButton.style.display = 'none';
+        const labelLitteral = itemLitteral.querySelector('ion-label');
+        if (labelLitteral) {
+          // If size of text > size of container
+          if (labelLitteral.scrollWidth > labelLitteral.clientWidth) {
```

Comment:
this seems relatively robust! good idea!

Reply:
Litteral -> Literal

Search and replace all occurences.